packages/connections/src/stores/connections-store.ts

```diff
@@ -363,6 +365,39 @@ export function useConnections(
           });
         }
       },
+      async removeConnection(connectionInfo: ConnectionInfo) {
+        await connectionStorage.delete(connectionInfo);
+        dispatch({
+          type: 'set-connections',
+          connections: connections.filter(
+            (conn) => conn.id !== connectionInfo.id
+          ),
+        });
+        const nextActiveConnection = createNewConnectionInfo();
+        dispatch({
+          type: 'set-active-connection',
+          connectionId: nextActiveConnection.id,
+          connectionInfo: nextActiveConnection,
```

Comment:
How do you feel about moving this setting of the active connection behind a check for if the connection being removed is the active one? I imagine if a user is trying to clean up some connections while editing a current connection they might not want to be moved to a new connection.